test/forces.jl

```diff
@@ -36,9 +37,55 @@ include("testcases.jl")
     diff_forces = dot(F1[1], disp)
     @test abs(diff_findiff - diff_forces) < 1e-7
 
-    # Rough test against QE reference (using PZ functional)
-    reference = [[-5.81108123960e-3, -4.60222477677e-3, -3.37528153911e-3],
-                 [ 5.81108123960e-3,  4.60222477677e-3,  3.37528153911e-3]]
+    # Test against QE v7.1 using LibXC v5.3.2 lda_x+lda_c_pw
+    # (see testcases_QuantumESPRESSO/silicon_LDA_forces)
+    reference = [[-5.809880257762980e-3, -4.601261807107919e-3, -3.374609176201532e-3],
+                 [ 5.809880257762980e-3,  4.601261807107919e-3,  3.374609176201532e-3]]
+    @test maximum(v -> maximum(abs, v), reference - Fc1) < 1e-5
+end
+
+@testset "Forces on silicon with non-linear core correction" begin
```

Comment:
Might also be worth refactoring the tests instead of copy n' paste (feel free to only put a todo here ... it's a bit of a mess anyway).